dangerzone/document.py

```diff
@@ -103,6 +103,31 @@ def default_output_filename(self) -> str:
 
     def announce_id(self) -> None:
         log.info(f"Assigning ID '{self.id}' to doc '{self.input_filename}'")
+    def set_output_dir(self, path: str) -> None:
+        if not self._output_filename:
+            self.set_default_output_filename()
+
+        # keep the same name
+        old_filename = os.path.basename(self.output_filename)
+
+        new_path = os.path.abspath(path)
+        if not os.path.exists(new_path):
+            raise errors.NonExistantOutputDirException()
```

Comment:
Not that important, but we start to have various errors for different aspects of a document (input file, output file, output dir), so I think it's best to start namespacing them:

* `InputFile...Exception`
* `OutputFile...Exception`
* `OutputDir...Exception`

Probably not actionable now, but something to have in mind.